GitUI/CommandsDialogs/FormCheckoutBranch.cs

```diff
@@ -484,9 +484,14 @@ private void Branches_SelectedIndexChanged(object sender, EventArgs e)
             {
                 ThreadHelper.FileAndForget(async () =>
                     {
-                        var currentCheckout = Module.GetCurrentCheckout();
+                        ObjectId? currentCheckout = Module.GetCurrentCheckout();
 
-                        Validates.NotNull(currentCheckout);
+                        // If there is no checkout yet, ahead/behind info is not applicable
+                        if (currentCheckout is null)
+                        {
+                            lbChanges.Text = "";
```

Comment:
Not on ui thread yet, await?

This could be handled in Module.GetCommitCountString(null,...) too

Reply:
> Not on ui thread yet

Thank you!

`Module.GetCommitCountString` is complex enough already.
I have rather turned into a single assignment.